reference-tests/src/test/java/net/consensys/linea/BlockchainReferenceTestTools.java

```diff
@@ -77,6 +77,14 @@ public class BlockchainReferenceTestTools {
     if (NETWORKS_TO_RUN.isEmpty()) {
       PARAMS.ignoreAll();
     }
+    // ignore tests that are failing because there is an account with nonce 0 and
+    // non empty code which can't happen in Linea since we are post LONDON only.
+    PARAMS.ignore("InitCollision_d0g0v0_London[London]");
```

Comment:
Yes, the three tests you have here are "impossible".  For the DafnyEVM, for example, we also had to mark them as ignored because they cannot arise in any meaningful way.  I don't actually understand what they are for.